concepts/50 React Components/02 Create a DevExtreme Application/40 Configure Themes/40 Apply Theme Variables to Custom Elements.md

```diff
@@ -0,0 +1,11 @@
+Theme variables are defined in the `src\themes\generated\variables.base.scss` and `src\themes\generated\variables.additional.scss` files. Apply them to custom elements so that the elements have a uniform appearance with the rest of the application.
```

Comment:
```suggestion
Theme variables are defined in the `src\themes\generated\variables.base.scss` and `src\themes\generated\variables.additional.scss` files. Apply them to custom elements, so that the elements have a uniform appearance with the rest of the application.
```